src/app.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * This file simply passes along your bot source and metadata to
+ * leftovers-again. No edits necessary.
+ */
+import * as LeftoversAgain from 'leftovers-again';
+import * as DeepIndigo from './bot';
+import * as metadata from '../package.json';
+export class X {}
```

Comment:
Didn't mean to leave this export, but it was because typescript was complaining that it wasn't a module, and we have `"isolatedModules": true`